### PR TITLE
Update Russian translation and fix one string

### PR DESCRIPTION
**Pull request by Shatur95**
* Update Russian translation
* Remove unnecessary space symbol from one string in settings

### CODE REVIEW

File: translations/Internationalization_ru.ts [hunk 1]
```diff
@@ -98,716 +98,706 @@ Use the Mouse Wheel to change the thickness of your tool.</source>
...
         <location filename="../src/config/configwindow.cpp" line="64"/>
         <source>Filename Editor</source>
-        <translation type="unfinished">Редактор имен файлов</translation>
+        <translation>Редактор имен файлов</translation>
     </message>
     <message>
         <location filename="../src/config/configwindow.cpp" line="69"/>
         <source>General</source>
-        <translation type="unfinished">Общее</translation>
+        <translation>Общее</translation>
```
Comment (sorrowless) [inline translations/Internationalization_ru.ts]: I think it should be renamed to "Общие", cause it's "Общие настройки", not "Общее хер знает что".

File: translations/Internationalization_ru.ts [hunk 2]
```diff
@@ -98,716 +98,706 @@ Use the Mouse Wheel to change the thickness of your tool.</source>
...
+        <translation>Скопировать выделение в буфер обмена</translation>
     </message>
 </context>
 <context>
     <name>DBusUtils</name>
     <message>
         <location filename="../src/utils/dbusutils.cpp" line="35"/>
         <source>Unable to connect via DBus</source>
-        <translation type="unfinished">Не удалось подключится через DBus</translation>
+        <translation>Не удалось подключится через DBus</translation>
```
Comment (sorrowless) [inline translations/Internationalization_ru.ts]: Подключится -> Подключиться

Что сделать? ПодключитЬся.

**Comment by ZetaoYang** [comment]
@Shatur95 Please submit the translated files to this [repo](https://github.com/flameshotapp/translations).
The Github pages of flameshot has been initially built. https://github.com/flameshotapp/flameshotapp.github.io

**Comment by Shatur95** [comment]
@ZetaoYang Sorry, I have missed it :)

**Comment by lupoDharkael** [comment]
Sorry to bother but the translations will be updated back in the main repository.
As indicated [here](https://github.com/lupoDharkael/flameshot/pull/147#issuecomment-370185605), there are no benefits in having an external repository.

**Comment by Shatur95** [comment]
@lupoDharkael, thanks for information!
Is I need to reupload translation to main repo?

**Comment by ZetaoYang** [comment]
@Shatur95 I am sorry for the inconvenience caused to you, it has been corrected to [translation-instruction](https://github.com/flameshotapp/translation-instruction). And it contains contributor list.
Thank you for your contribution. Please reopen this pr.

**Comment by Shatur95** [comment]
@ZetaoYang, sorry, I have already deleted rhis repo. I'll reupload translation in new PR.

**Pull request closed without merging**